src/scss/layout/_menu.scss

```diff
@@ -1,7 +1,7 @@
 [data-layout='menu'] {
-  align-items: center;
   display: flex;
   font-size: var(--small);
+  gap: var(--shim) var(--gutter-plus);
```

Comment:
Seems like we also need flex-wrap set here, in order for this to work as intended? Right now they stay side by side, and cause overflow on small screens.
![Screen Shot 2022-08-30 at 2 32 40 PM](https://user-images.githubusercontent.com/104161/187540800-de2183e3-d80f-4a95-af60-7acd02a3a051.jpg)